### ISSUE TITLE
Change also other components look

### ISSUE THREAD

**Issue by filips123** [os-js/osjs-windows8-theme#4]
Currently, only windows have look of Windows 8, but navbar is still on the top. It should also have Windows 8 look and should be at the bottom.

This is also for some other UI components and Windows 10 theme (#2).

**Comment by andersevenrud** [comment]
The panel is currently only moved upon user request, and default is always on top (but can be configured).

**Comment by filips123** [comment]
Shouldn't Windows themes use panel at bottom? As you said, it should be configured, but for Windows themes, bottom should be default.

Also, icons also need to be changed so they will look more like Windows.

**Comment by andersevenrud** [comment]
My thought here was that maybe swapping panel positions would be perceived as a bug. And if a user actually wants it on top, they'd then have to manually swap the position again. Which might be a bit bad UX --- not entirely sure yet, heh.

Icons are separate theme packs. There are `-theme`, `-sounds` and `-icons`.

**Comment by filips123** [comment]
I don't think so. Aren't themes meant to change design 🤔? Because Windows (by default) uses panel on the bottom, it should probably also be there in this theme.
It is also same for macOS (which is at bottom), Ubuntu GNOME (which is on the left) and other themes.

If this is explained in theme description (README) with also some screenshots, it won't be confusing.

Yes, and icons (and also maybe sounds) should be in separate package. And it should also be possible to use Windows 10 start menu style.
